docs/tutorials/standalone/Canopy/canopy_tutorial.jl

```diff
@@ -181,25 +181,8 @@ stomatal_model = MedlynConductanceModel{FT}(cond_params);
 
 # Arguments for photosynthesis model:
 
-photo_params = FarquharParameters{FT}(
-    Canopy.C3();
-    oi = FT(0.209),
-    ϕ = FT(0.6),
-    θj = FT(0.9),
-    f = FT(0.015),
-    sc = FT(5e-6),
-    pc = FT(-2e5),
-    Vcmax25 = FT(5e-5),
```

Comment:
This one is read in from a map and should not move to CP

Reply:
I will keep the default value in the constructor!

Reply:
Is that true for all the parameters that are staying here? (`sc` and `pc` too)

Reply:
`sc` and `pc` each have different values for some test cases in the repo, so I thought it would be best not to change them. They do have default values in climaparameters, so I could try removing the override values and see if the CI passes.
`sc` has values 5e-6 and 2e-6, `pc` has value -2e5 and -2e6

Reply:
It seems like they don't make a difference to the tests. There are still some constructors in ClimaLandSimulations that have hardcoded values, but they seem to have different requirements.